app/web/src/workers/webworker.ts

```diff
@@ -182,6 +185,22 @@ const start = async (sqlite3: Sqlite3Static, testing: boolean) => {
     );
   }
 
+  // Adding an integrity check for a corrupted SQLite
+  // db file. In the case it fails, delete the file
+  // and re-start this process
+  const integrity = sqlite.exec({
+    sql: "PRAGMA quick_check",
+    returnValue: "resultRows",
+  });
+  const ok = oneInOne(integrity);
+  if (ok !== "ok") {
+    log(`Integrity: failed`);
+    sqlite.close();
+    poolUtil?.unlink(`/${dbname}`);
+    await start(sqlite3, testing);
+    return;
+  }
```

Comment:
Repro steps:

1. start my web worker perf test
2. while its in the middle of running sql
3. refresh the browser tab
4. test fails to load b/c the file is corrupted
5. with this change instead you see
6. "Integrity: failed"
7. and the test starts up
8. Refresh again, no integrity message.